src/main/java/org/algorithms/IndexOfFirstOccurrenceInAString.java

```diff
@@ -0,0 +1,18 @@
+package org.algorithms;
+
+public class IndexOfFirstOccurrenceInAString {
+    public int strStr(String haystack, String needle) {
+        for(int i = 0; i < haystack.length(); i++) {
+            return haystack.indexOf(needle);
```

Comment:
Don't use ready method. Implement it yourself. Assume there is no `indexOf` ready.